### PR TITLE
Bookstore route setup

### PR DESCRIPTION
**Pull request by gids-dadi**
The PR is the basic setup of the book store app

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../style/Form.css';
+
+function form() {
```
Comment (zainsadaqat) [inline src/components/Form.js]: Please rename your "form component" as "CreateNewBook" to make it more meaningful and descriptive 👍🏻

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            placeholder="Enter book author"
+          />
+          <button type="submit">Add Book</button>
+        </form>
+      </>
+    </div>
+  );
+}
+
+export default form;
```
Comment (zainsadaqat) [inline src/components/Form.js]: Don't forget to match the component here as well 😄

File: src/components/Books.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Book from './Book';
+import Form from './Form';
```
Comment (zainsadaqat) [inline src/components/Books.js]: Don't forget to update the component name here as well 😄

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @gids-dadi  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @gids-dadi, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**